R/summary_functions.R

```diff
@@ -34,12 +40,11 @@ zero_coverage <- function(coverage_result){
 print.coverage <- function(x, ...) {
   df <- as.data.frame(x)
 
-  per_file_percents <-
```

Comment:
Now sure why this change was made.  With the S3 percent_defaults I mentioned above this can be simply
```r
per_file_percents <- tapply(df$value, df$filename, FUN = percent_coverage, simplify = TRUE)
```